cli/worker.rs

```diff
@@ -35,7 +35,7 @@ pub struct WorkerChannels {
 #[derive(Clone)]
 pub struct Worker {
   pub name: String,
-  isolate: Arc<Mutex<deno_core::Isolate>>,
+  isolate: Arc<Mutex<Box<deno_core::Isolate>>>,
```

Comment:
@piscisaureus Are you ok with forcing users of deno_core to use `Box<Isolate>` ?